migrations/etl/2_Mine_manager.sql

```diff
@@ -26,47 +26,35 @@ BEGIN
     SELECT count(*) FROM ETL_MANAGER into old_row;
 
     RAISE NOTICE '.. Update existing records with latest MMS data';
-    --1. Mine with valid manager attached
-    WITH now_manager AS (
-        SELECT
-            SUBSTRING(now_contact.cid from 1 for 7) AS mine_no      ,
-            contact_info.cid                        AS contact_cid  ,
-            contact_info.add_dt                     AS add_dt
-        FROM mms.mmsccc now_contact
-        INNER JOIN mms.mmsccn contact_info on
-            contact_info.cid=now_contact.cid_ccn
-        WHERE
-            SUBSTRING(now_contact.type_ind from 3 for 1) = 'Y'
+
+    --Step 1: Get regional mine_no's
+    WITH regional_mine AS (
+        SELECT mine_no
+        FROM mms.mmsmin
+        WHERE (min_lnk = 'N' OR min_lnk is null)
```

Comment:
Might make sense to say "not a major mine" `WHERE min_lnk != 'Y'` instead?

Reply:
Postgres is weird, that way results being excluded when min_lnk is null.

(https://stackoverflow.com/questions/36508815/not-equal-and-null-in-postgres).